### PR TITLE
Adding "Information Card" component to Storybook

### PR DESCRIPTION
**Pull request by mateusmarquezini**
## Hey, I just made a Pull Request!

- Note: This PR is still in progress, but if you wish, feel free to give me some feedback while I'm working on 😄 

- Added Information Card to the Storybook according to these specs: #400 

#### :heavy_check_mark: Checklist

- [x] All tests are passing `yarn test`
- [x] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [x] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] [CODEOWNERS](./CODEOWNERS) updated (for new stuff)
- [ ] Regression tests added for bug fixes

Default:

image

Subhead:
image

Link in footer:
image

### CODE REVIEW

File: packages/core/src/layout/InfoCard/BottomLink.js [hunk 1]
```diff
@@ -17,9 +17,24 @@
...
+  primary: {
+    fontWeight: '700',
+    color: grey[900],
+  },
+})(ListItemText);
+
+const ListItemBottom = withStyles({
+  root: {
+    maxWidth: 'fit-content',
+    padding: '16px 16px 16px 20px',
```
Comment (freben) [inline packages/core/src/layout/InfoCard/BottomLink.js]: For all spacing, make use of the theme; in this instance, `theme.spacing(2, 2, 2, 2.5)` (if we are using fractional spacing @Katz)

**Review by freben — COMMENTED** [review]
Great!

**Comment by stefanalund** [comment]
Looks great! 👌🏼

File: packages/core/src/layout/InfoCard/BottomLink.js (hunk 1 above)
Comment (mateusmarquezini) [reply packages/core/src/layout/InfoCard/BottomLink.js]: awesome! Thanks, Fredrik! :smile:

File: packages/core/src/layout/InfoCard/InfoCard.js [hunk 2]
```diff
@@ -29,7 +29,13 @@ import BottomLink from './BottomLink';
 
 import textContent from 'react-addons-text-content';
 
-const BoldHeader = withStyles({ title: { fontWeight: '700' } })(CardHeader);
+const BoldHeader = withStyles({
+  title: { fontWeight: '700' },
+  subheader: { paddingTop: '2px' },
+  root: {
+    padding: '16px 16px 16px 20px',
```
Comment (stefanalund) [inline packages/core/src/layout/InfoCard/InfoCard.js]: Same thing as above, please use `theme.spacing(2, 2, 2, 2.5)`

**Comment by stefanalund** [comment]
Maybe move this to proper PR now @mateusmarquezini ?

File: packages/core/src/layout/InfoCard/InfoCard.js (hunk 2 above)
Comment (mateusmarquezini) [reply packages/core/src/layout/InfoCard/InfoCard.js]: oops, my mistake. You're right. Fixed. Thx! :smile:

**Comment by mateusmarquezini** [comment]
> Maybe move this to proper PR now @mateusmarquezini ?

Yes @stefanalund ! I'm gonna commit the latest fixes

**Review by stefanalund — APPROVED** [review]

**Pull request merged**